templates/common/__test__/get.location.test.ts

```diff
@@ -0,0 +1,50 @@
+import assert from 'node:assert';
+import fs from 'node:fs';
+import { describe, it } from 'node:test';
+
+import { runTestFunction } from './function.helper.js';
+
+describe('get-location script template', () => {
+  it('should output workflow artifact location', (t) => {
+    const spy = t.mock.method(console, 'log');
+    const shimRequired: string[] = [];
+    runTestFunction(
+      './templates/common/get.location.yml',
+      [
+        {
+          toReplace: `JSON.parse(process.env['ARGO_TEMPLATE'])`,
+          replaceWith: JSON.stringify({
+            archiveLocation: {
+              s3: {
+                key: '2024-10/02-test-get-location-29l4x/test-get-location-29l4x-get-location-3125883809',
+                bucket: 'linz-workflows-scratch',
+              },
+            },
+          }),
+        },
+      ],
+      (req) => {
+        shimRequired.push(req);
+        if (req === 'node:fs') return fs;
+        throw new Error('Failed');
```

Comment:
Should the first line also be conditional on `req === 'node:fs'`? If so, maybe an early exit might be easier to read:
```suggestion
        if (req !== 'node:fs') throw new Error('Failed');
        shimRequired.push(req);
        return fs;
```